.circleci/config.yml

```diff
@@ -271,6 +274,12 @@ jobs:
           command: |
             IMG="image-$(date +%j).gz"; docker save addons-frontend | pigz --fast -c > ~/addons-frontend/docker/$IMG; ls -l ~/addons-frontend/docker
             ls -l ~/addons-frontend/docker
+      - run:
+          name: Check image works
+          command: |
+            docker run --rm -p 4000:4000 -d -e NODE_ENV=production -e NODE_CONFIG_ENV=prod addons-frontend
```

Comment:
I wonder if (perhaps) a simple docker compose file with a healthcheck would achieve this in a more reproducible way without adding much complexity.

```yaml
services:
  addons-frontend:
    image: IMAGE_NAME
    environment:
      - NODE_ENV=production
      - NODE_CONFIG_ENV=prod
    ports:
      - "4000:4000"
    healthcheck:
      test: [
        "CMD-SHELL",
        "curl -f http://127.0.0.1:4000/__frontend_lbheartbeat__"
      ]
      interval: 10s
      timeout: 5s
      retries: 10
```

Maybe that could work? Now you can just run

```bash
docker compose up --wait -d
```

Could be a follow up

Reply:
That's an option we could look into as a follow-up yeah. I didn't want to introduce docker compose into the mix since we don't currently use it when running the frontend locally (and we're already using manual `docker` commands in the CircleCI config)